### ISSUE TITLE
State of the project

### ISSUE THREAD

**Issue by repli2dev** [bckp/roadrunner#1]
Is this project dead? Recent fork & no released versions and quite low on docs suggests this.

**Comment by bckp** [comment]
Not really, just old company that use this was bought and killed, so right now, we are trying to resurrect this under new company, we belive in 2025 will bring some nice changes we were playing with

**Comment by delacry** [comment]
If you are interested, you can force push https://github.com/Rixafy/roadrunner/tree/php84 or I can create a PR, but there is some failing test with sending headers early and I'm not using a lot of features (I had some problems maybe a year ago in real-world application with sessions etc.), I'm using it on just a simple console project that manages some services on the server, and so far it's working alright.
